packages/stylelint-config/__tests__/index.test.js

```diff
@@ -1,9 +1,10 @@
-const fs = require('fs');
```

Comment:
Oh damn so wait are tests ESM now?

Reply:
Yeah though vitest ships a CJS file for tests that use only `require()` it's actually empty[^1] so if you do `const { it, expect } = require('vitest')` you'll get a "it is not a function" error because the export from `vitest` that's being loaded there is just an empty object.

Kinda annoying but cool that all of these tests can use `import` without any TS or Babel configs.

[^1]: https://github.com/vitest-dev/vitest/blob/main/packages/vitest/index.cjs

Reply:
Gotcha! I prefer this pattern of importing these rather than the mysterious jest global shit